msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -0,0 +1,381 @@
+
+
+- title: Set up app settings and manifest files
+  durationInMinutes: 1
+  content: |
+    1. Navigate to **appsettings.json** in cloned repository.
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/appsettingslocation.png" alt-text="App settings location" border="true":::
+
+    1. Open **appsettings.json** in **Visual Studio 2019** and update the following information:  
+
+         * Set `"MicrosoftAppId"` to your bot's **Microsoft App ID**.
+         * Set `"TenantID"` to **Directory (tenant) ID**.
+         * Set `"MicrosoftAppPassword"` to your bot's client secret ID value.
+         * Set `"BaseUrl"` to the `[WebAppDomainName]`.
+
+      
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/app-settings-details.png" alt-text="Appsettings" border="true":::
+
+    1. Navigate to **manifest.json** in cloned repository.
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/manifestlocationpackage.png" alt-text="Manifest location" border="true":::
+
+    1. Open **manifest.json** in latest version of Visual Studio and make the following changes:
+
+         * Replace all occurrences of `<<Microsoft-App-ID>>` with your bot's **Microsoft App ID**.
+         * Replace all occurrences of `<<Base_URL>>` with your fully qualified domain name.
+         * Set `validDomains` to `[WebAppDomainName]`.
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/manifestfile.png" alt-text="Manifest file" border="true":::
+
+- title: Build and run the service
+  durationInMinutes: 1
+  content: |
+    **To build and run the service using Visual Studio
+
+    # [Latest version of Visual Studio](#tab/vs2022)
+
+       1. Open the latest version of Visual Studio.
+       1. Navigate to **File** > **Open** > **Project/Solution**.
+    
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/VSopenfile.png" alt-text="Open file in Visual Studio" border="true":::
+
+       1. Select **ActivityFeedBroadcast.sln** file from **csharp** folder.
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/openproject.png" alt-text="Open a project" border="true":::
+
+       1. Press **F5** to run the project.
+    
+       1. Select **Yes** if the following dialog appears:
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/graphactivity-generator-certificate.png" alt-text="Select certificate" border="true":::
+
+          A webpage opens with a message **Your bot is ready!**
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/appisready.png" alt-text="App is ready" border="true":::
+
+   
+- title: Add activity feed broadcast app to Teams
+  durationInMinutes: 1
+  content: |
+    **To add activity feed broadcast app to Teams**
```

Comment:
We can remove this sentence, it's the same information as the title.

Reply:
Comments incorporated